packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -278,6 +278,9 @@ fi
 %endif
 
 %changelog
+* Tue Jun 02 2020 Meghna Srivastav <mssrivas@amazon.com> - 1.40.0
```

Comment:
should be "1.40.0-1", same for `packaging/ubuntu-trusty/debian/changelog`, `packaging/suse/amazon-ecs-init.changes` and `scripts/changelog/CHANGELOG_MASTER`